virt.if

```diff
@@ -1539,6 +1539,24 @@ interface(`virt_rw_svirt_image',`
 	allow $1 svirt_image_t:file rw_file_perms;
 ')
 
+########################################
+## <summary>
+##	Read and write to svirt_image sockfiles.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`virt_write_svirt_image_sock_file',`
+	gen_require(`
+		type svirt_image_t;
+	')
+
+	write_sock_files_pattern($1, svirt_image_t, svirt_image_t)
```

Comment:
Do we know which objects are labeled as svirt_image_t ? I'm not saying it's wrong, just want to understand if the svirt_image_t for sockets is good approach.